src/main/java/world/bentobox/bentobox/api/commands/CompositeCommand.java

```diff
@@ -51,6 +45,17 @@
      * True if the command is for the player only (not for the console)
      */
     private boolean onlyPlayer = false;
+    
+    /**
+     * True if the command is only for the console to use
+     */
+    private boolean onlyConsole = false;
```

Comment:
I don't have a single clue about any command that might one day need it, but, well, let's keep it :D

Reply:
I was thinking of some commands for BentoBox that would show a lot of text or stats about the addons. It's just in case though.